src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -121,5 +121,16 @@
         "kafkatrigger",
         "kafka"
     ]
+  },
+  {
+    "id": "Microsoft.Azure.WebJobs.Extensions.WebPubSub",
+    "version": "1.0.0-beta.3",

```

Comment:
can you change this to `"majorVersion": "1",`

Reply:
updated.